Bottom: 29c49ec344b03ddd9b5ec838c16bd55c0982ce06
Top:    59c376c697def8496ef9c517b72403d48f835bc0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-07 11:45:12 +0000

Refresh of polygon1-c-move-jump-call-to

---

diff --git a/src/polygon1.c b/src/polygon1.c
index ede09eb..cb3590e 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -2459,7 +2459,7 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
 	break;
 #endif
 
-#if 1
+#if 0
       /* see where to go next */
       if (!jump (&curv, &dir, j_rule))
 	break;
@@ -2536,9 +2536,9 @@ Gather (VNODE *startv, PLINE **result, J_Rule j_rule, DIRECTION initdir)
 
       /* Advance to the next vertex (edge?).  */
       curv = (dir == FORW) ? NEXT_VERTEX (curv) : PREV_VERTEX (curv);
-#if 0
+#if 1
       /* see where to go next */
-      if (!jump (&cur, &dir, j_rule))
+      if (!jump (&curv, &dir, j_rule))
 	break;
 #endif
     }
